docs/aws-lambda-extension.asciidoc

```diff
@@ -21,16 +21,19 @@ By using a custom-built AWS Lambda Extension, Elastic APM Agents can send data t
 
 The rest of this guide contains instructions for instrumenting a Lambda function. There are two high level steps to instrumenting an AWS Lambda function.
 
-1. Install the Lambda extension
-2. Configure the Agent and Lambda function handler
+1. Installing the Lambda Extension ...
+   * ... <<aws-lambda-install,Using the Installer>>
+   * ... <<aws-lambda-manual-instrumentation,Through Manual Installation>>
```

Comment:
Why the ellipsis here?

Reply:
cc @AlexanderWert - is this a placeholder for something?